src/main/java/com/foundation/salesforce/entities/Opportunity.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * @(#) Opportunity.java Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package com.foundation.salesforce.entities;
+
+/**
+ * Opportunity class
+ *
+ * @author Cristian Lujan
+ * @version 1.0
```

Comment:
Please put point at the end the version.

Reply:
I don't think the version has to end up with spaces.